guacamole/src/main/frontend/src/app/player/directives/player.js

```diff
@@ -179,6 +199,12 @@ angular.module('player').directive('guacPlayer', ['$injector', function guacPlay
          */
         var resumeAfterSeekRequest = false;
 
+        /**
+         * A scheduled timer to clean up the mouse activity CSS class, or null
+         * if no timer is scheduled.
+         */
+        var mouseActivityTimer = null;
```

Comment:
I don't see anything assigning a value to this variable. Is this used?

Reply:
Oops, yeah, I lost the assignment when I refactored. let me fix that.

Reply:
Oops, yeah, I lost the assignment when I refactored. let me fix that.

Reply:
Fixed.